src/soter/openssl/soter_ec_key.c

```diff
@@ -27,6 +27,7 @@
 
 #include "soter/soter_portable_endian.h"
 
+#ifndef THEMIS_EXPERIMENTAL_OPENSSL_3_SUPPORT
```

Comment:
the common behavior is to use positive conditions. lets use first statement for the positive condition and the negative in the "else" statement everywhere. it much easier to read